lib/wibox/layout/stack.lua

```diff
@@ -51,16 +51,23 @@ local stack = {mt={}}
 -- @name remove_widgets
 -- @class function
 
---- Add spacing between each layout widgets
+--- Add spacing around the widget, similar to the margin container.
+--@DOC_wibox_layout_stack_spacing_EXAMPLE@
```

Comment:
Just as this the previous PRs, I really don't think it makes any difference. It render fine if the file doesn't exists and reduce patch noise from the test commit,